modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -36,10 +38,37 @@ metadata:
 spec:
   sso:
     provider: keycloak
+    keycloak:
+      rootCA: "<PEM-encoded-root-certificate>" <1>
   server:
     route:
-     enabled: true
+      enabled: true
+----
+<1> A custom certificate used to verify the Keycloak's TLS certificate.
++ 
+The Operator reconciles changes in the `.spec.keycloak.rootCA` field and updates the `oidc.config` in `argocd-cm` configuration map with the PEM encoded root certificate.
+
+[NOTE]
+====
+To proceed with an insecure connection, leave the value of `rootCA` empty and use `skip.verify` as shown below:
+
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
+spec:
+  extraConfig:
+    oidc.tls.insecure.skip.verify: "true"
+  sso:
+    provider: keycloak
+    keycloak:
+      rootCA: ""
 ----
+====
 
 [NOTE]
```

Comment:
Can we make this note part of the running text ? because in the preview we now have 2 notes in continuation, which does not actually go with the flow.